src/domains/pnpg-common/01_key_vault_operations.tf

```diff
@@ -0,0 +1,86 @@
+# JWT
+module "jwt" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//jwt_keys?ref=v5.3.0"
+
+  jwt_name         = "jwt"
+  key_vault_id     = module.key_vault_pnpg.id
+  cert_common_name = "apim"
+  cert_password    = ""
+  tags             = var.tags
+}
+
+module "jwt_exchange" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//jwt_keys?ref=v5.3.0"
+
+  jwt_name         = "jwt-exchange"
+  key_vault_id     = module.key_vault_pnpg.id
+  cert_common_name = "pnpg.selfcare.pagopa.it"
+  cert_password    = ""
+  tags             = var.tags
+}
+
+module "agid_spid" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//jwt_keys?ref=v5.3.0"
+
+  jwt_name         = "agid-spid"
+  key_vault_id     = module.key_vault_pnpg.id
+  cert_common_name = "pnpg.selfcare.pagopa.it"
+  cert_password    = ""
+  tags             = var.tags
+}
+
+resource "null_resource" "upload_jwks" {
+  triggers = {
+    "changes-in-jwt" : module.jwt.certificate_data_pem
+    "changes-in-jwt-exchange" : module.jwt_exchange.certificate_data_pem
+  }
+  provisioner "local-exec" {
+    command = <<EOT
+              mkdir -p "${path.module}/.terraform/tmp"
+              pip install --require-hashes --requirement "${path.module}/utils/py/requirements.txt"
```

Comment:
maybe you could re-use the scripts already wrote for selfcare instead of copy/paste